privacy-protections/storage-blocking/main.js

```diff
@@ -0,0 +1,411 @@
+const THIRD_PARTY_ORIGIN = 'https://good.third-party.site';
```

Comment:
third party that we are using for a 3p cookie test and 3p storage via iframe test (this is the domain that should *not* be blocked)